pkg/controllers/v1alpha1/datamigrate/datamigrate_controller.go

```diff
@@ -92,6 +97,7 @@ func (r *DataMigrateReconciler) SetupWithManager(mgr ctrl.Manager, options contr
 	return ctrl.NewControllerManagedBy(mgr).
 		WithOptions(options).
 		For(&datav1alpha1.DataMigrate{}).
+		Owns(&batchv1.CronJob{}).
```

Comment:
Shall we better add a labelSelector in `manager.Cache` to force the datamigrate controller cares about some specific cronjob?

Reply:
@zwwhdls could you please create a new PR to enhance it?